README.md

```diff
@@ -141,3 +163,57 @@ bower install
 # Run Grunt to build the interface
 grunt
 ```
+
+## Deployment Steps
+
+1. Follow the steps to [edit settings in config.py as described
+   above](#edit-settings-in-configpy-file).
+1. Fork the [deployment repository](TODO: migrate repo to github) and then clone
```

Comment:
Thanks for the reminder. I just completed the migration and filled in that TODO.